test/Microsoft.ML.CpuMath.UnitTests.DotNetStandard/UnitTests.cs

```diff
@@ -0,0 +1,450 @@
+using System;
```

Comment:
Instead of duplicating this file in the 2 different folders, we should be linking both .csproj files to the same physical .cs file.  That way we only need to maintain a single file.

Reply:
This is how you do something like that.  The file can live in one of the folders, and the other `.csproj` can do something like:

`<Compile Include="..\Microsoft.ML.Core\Utilities\Contracts.cs" />`